README.md

```diff
@@ -106,18 +142,20 @@ specified in the table's metadata as well.
   export TAP_SNOWFLAKE_WAREHOUSE=<snowflake-warehouse>
 ```
 
-2. Install python dependencies in a virtual env and run nose unit and integration tests
+NOTE: User must have permission to create/drop schema (TAP_SNOWFLAKE_TEST), create/drop table, insert into those tables
```

Comment:
Referring to my earlier note - saw this was updated. Maybe just make it clear which DB we are using in Snowflake for this.

Reply:
Its DATA_DEG_TEST.TAP_SNOWFLAKE_TEST. I can add it to the readme